packages/connectivity/src/scp-cf/destination/get-subscriber-token.ts

```diff
@@ -92,3 +101,63 @@ function getJwtForServiceToken(
     return options.jwt;
   }
 }
+
+/**
+ * @internal
+ * Get a subscriber token pair with required fields. Checks that at least one of the tokens exists and sets defaults if needed.
+ * @returns The decoded subscriber tokens.
+ */
+export function getRequiredSubscriberToken(
+  token: SubscriberToken | undefined
+): Required<SubscriberToken> {
+  if (token) {
```

Comment:
style: any reason not to do an early return here which would allow the rest of the function not to be inside the scope of the if?

```
if (!token) {
 throw error
}

// rest of the function
```

Reply:
I'd have to throw the error twice.